serverlessworkflow/modules/ROOT/pages/core/custom-functions-support.adoc

```diff
@@ -417,11 +417,42 @@ Is exactly the same as:
   ]
 ----
 
+[[sending-cloudevents]]
+==== Sending CloudEvents
+
+By default, {product_name} sends the payload of a Knative function as a regular JSON object and `Content-Type` as `application/json`. However, you can tell {product_name} to send the payload as a CloudEvent. In that case, {product_name} will check if the CloudEvent has all mandatory attributes set and use `application/cloudevents+json; charset=UTF-8` in `Content-Type`.
+
+To tell {product_name} you want to send the payload as a CloudEvent, you must define your function like:
+
+[source,json]
+----
+  "functions": [
+    {
+      "name": "greet",
+      "type": "custom",
+      "operation": "knative:custom-function-knative-service",
+      "metadata": {
+        "path": "/",
+        "asCloudEvent": true <1>
+      }
+    }
+  ]
+----
+
+<1> Tells {product_name} to send the payload as a CloudEvent. Default is `false`
+
+[WARNING]
+====
+You can't send a CloudEvent if you don't set the `asCloudEvent` attribute to `true`. If you do so, {product_name} will throw an error.
+====
+
```

Comment:
This should be merged with the `Sending a CloudEvent` section. I don't see any benefit having it split in two places.

Reply:
Done